day00/ex07/ex07.md

```diff
@@ -0,0 +1,56 @@
+# Exercise 07 - Mean Squared Error
+
+|                         |                    |
+| -----------------------:| ------------------ |
+|   Turn-in directory :   |  ex07              |
+|   Files to turn in :    |  mse.py            |
+|   Forbidden functions : |  None              |
+|   Remarks :             |  n/a               |
+
+## Objectives:
+
+You must implement the following formula as a function:
```

Comment:
here also, maybe explain the purpose of mean square error ? (remark for future enhancement purpose)